src/metrics/chaoss.ts

```diff
@@ -358,3 +358,62 @@ FORMAT JSONCompact`;
     }
   });
 };
+
+interface IssueAgeOptions {
+  type: 'avg' | 'median';
+  unit: 'week' | 'day' | 'hour' | 'minute';
+}
+export const chaossIssueAge = async (config: QueryConfig<IssueAgeOptions>) => {
+  config = getMergedConfig(config);
+  const whereClauses: string[] = ["type = 'IssuesEvent'"];
+  const repoWhereClause = getRepoWhereClauseForClickhouse(config);
+  if (repoWhereClause) whereClauses.push(repoWhereClause);
+
+  const endDate = new Date(`${config.endYear}-${config.endMonth}-1`);
+  endDate.setMonth(config.endMonth);
+  
+  let type = filterEnumType(config.options?.type,['avg','median'],'avg');
+  let unit = filterEnumType(config.options?.unit,['week','day','hour','minute'],'day');
+
+  const sql = `
+SELECT 
+  id, 
+  argMax(name, time) AS name,
+  ${getGroupArrayInsertAtClauseForClickhouse(config, {key:`issue_age`, defaultValue: 'NaN'})}
+FROM
+(
+  SELECT
+    ${getGroupTimeAndIdClauseForClickhouse(config, 'repo')},
```

Comment:
The `getGroupTimeAndIdClauseForClickhouse ` functions has another parameter which is the name of the time column. It will use `created_at` by default if not set which covers most situation.

So you should call `getGroupTimeAndIdClauseForClickhouse(config, 'repo', 'opened_at')` since the inner select table has no `created_at` column and issues should group by open time.